components/family-studies/api/src/main/java/org/phenotips/studies/family/internal/PhenotipsFamilyRepository.java

```diff
@@ -154,7 +154,7 @@ public synchronized boolean deleteFamily(Family family, User updatingUser, boole
         try {
             XWikiContext context = this.provider.get();
             XWiki xwiki = context.getWiki();
-            xwiki.deleteDocument(xwiki.getDocument(family.getDocument(), context), context);
+            xwiki.deleteDocument(xwiki.getDocument(family.getXDocument(), context), context);
```

Comment:
same change [here](https://github.com/phenotips/phenotips/blob/PT-2860/components/patient-data/rest/src/main/java/org/phenotips/data/rest/internal/DefaultPatientByExternalIdResourceImpl.java#L173) and [here](https://github.com/phenotips/phenotips/blob/PT-2860/components/patient-data/rest/src/main/java/org/phenotips/data/rest/internal/DefaultPatientResourceImpl.java#L175) ?